python/tvm/relay/frontend/tensorflow.py

```diff
@@ -2835,7 +2835,7 @@ def _parse_param(self, key, value, name, shape):
 
             array_ndim = len(np_array.shape)
             if array_ndim == 0:
-                self._nodes[name] = [tvm.relay.const(np_array)]
+                self._nodes[name] = [tvm.relay.const(np_array, np_array.dtype)]
```

Comment:
Do we also need to fix non-scalar case？

Reply:
For non-scalar cases, it goes to self._params[name] = tvm.nd.array(np_array).
Seems the tvm.nd.array function doesn't have a default dtype, it always gets dtype from np_array. While tvm.relay.const has a default param dtype set to None.